salt/modules/slack_notify.py

```diff
@@ -166,15 +166,22 @@ def find_user(name, api_key=None):
     return False
 
 
-def post_message(channel, message, from_name, api_key=None, icon=None):
+def post_message(
+    channel, message, from_name, api_key=None, icon=None, attachments=None, blocks=None,
+):
     """
     Send a message to a Slack channel.
 
+    .. versionchanged:: 3002.2
```

Comment:
Actually, what should this version be set to?

Reply:
3003